test/hotspot/jtreg/runtime/ErrorHandling/TestSigInfoInHsErrFile.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
There are no longer any changes to this file in this patch so the
copyright year update should be backed out.